lingua_franca/format.py

```diff
@@ -370,6 +374,39 @@ def nice_date_time(dt, lang='', now=None, use_24hour=False,
                                              use_ampm)
 
 
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+def nice_day(dt, date_format='MDY', include_month=True, lang=""):
+    if include_month:
+        month = nice_month(dt, date_format, lang)
+        if date_format == 'MDY':
+            return "{} {}".format(month, dt.strftime("%d"))
+        else:
+            return "{} {}".format(dt.strftime("%d"), month)
+    return dt.strftime("%d")
+
+
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+def nice_weekday(dt, lang=""):
+    if lang in date_time_format.lang_config.keys():
+        localized_day_names = list(
+            date_time_format.lang_config[lang]['weekday'].values())
```

Comment:
There has to be a `cache` call upon `date_time_format` to be safe to say that the keys are there.

I'm inclined to rewrite the class, that poor class is crammed
object dict created on init, with a helper `get_date_time_format(lang)`